pynsee/utils/init_conn.py

```diff
@@ -24,12 +20,12 @@ def opener(path, flags):
 
 
 def init_conn(
-    sirene_key: str, http_proxy: str = "", https_proxy: str = ""
+    sirene_key: str = None, http_proxy: str = None, https_proxy: str = None
```

Comment:
these needs to be ``str | None``, then, but it's not consistent with your test for sirene_key == "" above, we should pick one or the other

Reply:
Yes, that would be the same anyway for sirene_key. Again, the aim here was to force an update of credentials when setting an empty string for http_proxies (= back to home network with my laptop) which is quite different from http_proxies not being set.

Reply:
> str | None
Just found why I had'nt ever used that syntax... It appeared in python 3.10+ 😄

Reply:
Ow, sorry, thought it was 3.9... Might bad, we'll have to stick with ``Optional``, then